src/main/java/uk/gov/hmcts/reform/cpo/data/CasePaymentOrderEntity.java

```diff
@@ -17,6 +18,7 @@
 @Setter
 @Entity
 @Table(name = "case_payment_orders")
+@Audited(withModifiedFlag = true)
```

Comment:
Nice 👌

Reply:
Any business requirement for "querying revisions of entity that modified given property" ? otherwise 'withModifiedFlag' flag comes with some cost.

Reply:
https://tools.hmcts.net/jira/browse/CPO-16 states

`Note that we want to use @Audited( withModifiedFlag=true ) to enable tracking which properties of audited entities were modified in a given revision.`

Reply:
@mario-paniccia  ^^

Reply:
No business requirement for that, it's true. But given that I didn't expect the amount of changes to CPOs to be that big I thought it won't hurt to use it. 
But if we feel we want to apply YAGNI, ok for me to remove it. As you prefer